wiki/Daily_challenge/en.md

```diff
@@ -0,0 +1,65 @@
+# Daily Challenge
+
+The **daily challenge** is a multiplayer mode in osu!(lazer) where players are able to build up a *daily* streak by passing beatmaps consecutively with increasing difficulty.
+
+Each day will introduce one new map from the [featured artist](/wiki/People/Featured_Artists) listing with increased difficulty. Regularly after one week however, the difficulty would be reset again.
+
+Sometimes, the maps could come with a mod forced onto users, which required them to pass the map with the selected mod.
+
+## Game menu
+
+*Notice: Daily challenge requires an [osu! account](/wiki/Registration), and is not available for [silenced](/wiki/Silence) players.*
+
+From the main menu, the daily challenge menu can be accessed with the following steps:
+
+1. Click the `Play` button or press `P`.
+2. Click the `Daily challenge` button or press `D`.
```

Comment:
Not rly sure why the users would need a guide on how to find the daily challenge. The article already makes it clear that it's a lazer only mode so people won't look for it in stable. I don't think we need to tell them any further as the daily challenge can't be missed when you look in lazer and if it can, then the design in lazer should be changed since something like the daily challenge should not be missable.

Reply:
I based this article based on the `Multiplayer` one, so i decided to add this. I also kept this because i personaly didn't even know that the main menu has keyboard shortcuts and i discovered them thanks to the outdated `Multiplayer` article which has them. 

I don't think this mini guide needs to be deleted, but if more people will want this change then ill drop the mini guide

Reply:
The keyboard shortcuts are a useful addition in my opinion.